src/utils/cplay.c

```diff
@@ -67,6 +67,7 @@
 #include <getopt.h>
 #include <sys/time.h>
```

Comment:
@LaurentiuM1234 please update the commit message to reflect the corect 'key' used for suspend/resume.

make sure you also add 'Enter' as a possible key.